lib/openscap_parser/profiles.rb

```diff
@@ -1,25 +1,21 @@
 # frozen_string_literal: true
-require 'openscap_parser/test_result'
 
 module OpenscapParser
   # Methods related to saving profiles and finding which hosts
   # they belong to
   module Profiles
-    include TestResult
```

Comment:
No need for these mixins to be coupled so tightly IMO

Reply:
:+1: